client/src/stories/1-LoginInput.stories.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import LoginInput from '../components/LoginInput';
+
+export default {
+  title: 'LoginInput',
+};
+
+export const Default = () => (
+  <LoginInput width="280px" height="66px" placeholder="example@example.com" />
```

Comment:
It's not a good practice to set CSS in HTML/React properties.
The size of the input fields is usually set by the form/container (e.g. flexbox).

Same for the `Button`.

If you like to demonstrate how this `input` looks like in a container, you could add the container in the story.
See https://storybookjs.netlify.com/official-storybook/?path=/story/basics-form-input--sizes

Reply:
Right. I remember you saying that 1-10 times 🙈